bundles/core/org.eclipse.smarthome.core.thing/src/main/java/org/eclipse/smarthome/core/thing/internal/BridgeImpl.java

```diff
@@ -24,7 +24,10 @@
 public class BridgeImpl extends ThingImpl implements Bridge {
 
     private transient List<Thing> things = new CopyOnWriteArrayList<>();
-    private final Logger logger = LoggerFactory.getLogger(BridgeImpl.class);
+
+    private static Logger logger() {
```

Comment:
Just for my understanding: What would be the problem to simply add the transient keyword to the existing logger declaration?